lib/modules/filteReddit/Filterline.js

```diff
@@ -62,6 +63,9 @@ export default class Filterline {
 								<span>Hide currently filtered</span>
 							</label>
 						</div>
+						<div class="res-filterline-show-help">
+							Usage information
```

Comment:
Probably superfluous at this stage, as the help tip nor any of the Filterline stuff is translated yet.